examples/nats-client-secure-connection/nats-client-secure-connection.md

```diff
@@ -1,7 +1,6 @@
 # NATS client - SSL/TLS
 
-The NATS client connections can be secured by encrypting with TLS. In this example, the underlying connection of the listener is secured with basic authentication.
-In this example, the underlying connection of the publisher is secured with TLS/SSL. 
+The NATS server uses TLS semantics to encrypt client, route, and monitoring connections. TLS can be used to encrypt traffic between client/server and check the server’s identity. In this example, the underlying connection of the client is secured with TLS/SSL. A secured NATS client is created by passing the URL of the NATS broker and providing the TLS/SSL details using the `nats:SecureSocket` record. Use it to connect to a NATS server which has SSL/TLS enabled.
```

Comment:
```suggestion
The NATS server uses TLS semantics to encrypt the client, route, and monitor the connections. TLS can be used to encrypt traffic between the client/server and check the server’s identity. In this example, the underlying connection of the client is secured with TLS/SSL. A secured NATS client is created by passing the URL of the NATS broker and providing the TLS/SSL details using the `nats:SecureSocket` record. Use it to connect to a NATS server, which has SSL/TLS enabled.
```
Check if the technical meaning gets affected when saying `monitor` in the first sentence.